lib/Mojo/Promise.pm

```diff
@@ -41,6 +41,31 @@ sub finally {
   return $new;
 }
 
+sub map {
+  my ($class, $options) = (shift, ref $_[0] eq 'HASH' ? shift : {});
+  my ($cb, @items) = @_;
+
```

Comment:
There are no tests for empty `@items` and no promise returning check, e.g.
`return $class->reject('nothing to map') unless @items;`

Reply:
As this just delegates to Mojo::Promise::all I assumed that this would just be a pass-through.